src/components/Header.tsx

```diff
@@ -5,12 +5,12 @@ import { Link } from 'react-router-dom';
 export default () => {
   return (
     <header className={styles.header}>
-      <div>ICULive</div>
+      <div class={styles.nav}>ICULive</div>
```

Comment:
React uses syntax embedded in JavaScript, and `class` is a keyword in JavaScript. For the `class` attribute, React makes an exception and requires calling it `className`.